rpa-builder/6.7/modules/ROOT/pages/toolbox-excel-operations-execute-excel-macro.adoc

```diff
@@ -0,0 +1,66 @@
+= Execute Excel Macro 
+
+Run a macro in an Excel file or sheet. You can specify and pass up to 30 variables to the macro.
+
+Do not include this action step inside an Excel Session, Execute Excel Macro runs independently. 
+
+Execute Excel Macro requires Microsoft Excel installed on the computer that runs the automation. Otherwise, the action step fails to execute. 
+
+== Before You Begin
+
+Before using this action step, ensure that: 
+
+* Microsoft Excel is installed on the computer that runs the automation
+
+== Properties
```

Comment:
I wonder if these should be tables since they're reference content.

Reply:
Make sure all descriptions are parallel and remove periods if none are complete sentences.